### DISCUSSION TITLE
Is Rustdesk app still avalaible in the Google Play Store ?

### DISCUSSION THREAD

**Discussion by Philippe34** [rustdesk/rustdesk#5660] [Q&A] [answered]
I had installed the application on my Android phone, but I had to uninstall it.
I would like to reinstall it now, but it is no longer available. Do you also notice this disappearance?

**Comment by dinger1986** [comment]
yes, please reinstall directly from github, apk is available here https://github.com/rustdesk/rustdesk/releases/tag/1.2.2

**Reply by msymeonides** [reply]
Which package is the right one for a Samsung Galaxy S21?

**Reply by jpmahowald** [reply]
To second what was said in another thread, I recommend the F-Droid store: https://f-droid.org/packages/com.carriez.flutter_hbb/

F-Droid app is easy to install the right build for you phone, and keep it updated and secure. Their volunteer creators will make sure the one result when searching "rustdesk"  is from official sources.

Technically, this is a different group doing those builds. Which is a healthy thing, and should result in the same rustdesk. Only gets tricky in corner cases, like if you want the very latest features that aren't in a tagged release yet.

**Comment by Philippe34** [comment]
@dinger1986 
Thank you for your answer.
Il will try to do like this, but it is not so friendly than Play Store.

**Reply by dinger1986** [reply]
no it isnt but it will come back

**Reply by ximex** [reply]
is there an ETA?

**Reply by kolAflash** [reply]
I'd be interested in the background!

Why did RustDesk disappear from PlayStore?  
Did Google remove RustDesk from PlayStore or did the RustDesk maintainers remove it themselves?  
If Google: Can you share what Google requires RustDesk to change to return to PlayStore?

PlayStore (not available 2023-09-15):  
https://play.google.com/store/apps/details?id=com.carriez.flutter_hbb

F-Droid Store (still available):  
https://f-droid.org/packages/com.carriez.flutter_hbb/  
_Hint: Install F-Droid first -> https://f-droid.org/F-Droid.apk (alternative Android AppStore)_

**Comment by idofilus** [comment]
Seems like scammers using it to steal from users

https://support.google.com/googleplay/thread/224563444/rustdesk-app?hl=en
https://support.google.com/googleplay/thread/232808859
https://support.google.com/googleplay/thread/204405225/is-it-possible-for-an-app-on-play-store-to-have-malware?hl=en

Fkthem

**Comment by rustdesk** [comment, marked as answer]
https://twitter.com/rustdesk/status/1698372220379349421?t=EwtnYWnA98Yfw5kGiQXrGQ&s=19

or F-droid, https://f-droid.org/de/packages/com.carriez.flutter_hbb/

**Reply by kolAflash** [reply]
I already assumed it's because of scammers :-/  
And I absolutely understand that you temporarily unpublished RustDesk from Google Play. Let's hope a good solution can be found.

Does anyone know how other Android apps like [TeamViewer QuickSupport](https://play.google.com/store/apps/details?id=com.teamviewer.quicksupport.market) or [AnyDesk Remote-Desktop](https://play.google.com/store/apps/details?id=com.anydesk.anydeskandroid) handle scammers?  
Maybe a better warning mechanism is needed when giving remote access to someone else?  
Something like the warning text on the https://www.rustdesk.com/ website?

... oh I see there's already work being done :-)  
#5664 _Add mobile scam warning window_

See also: #1675 _Warning pop to make scams a little harder_

&nbsp;

&nbsp;

By the way:  
I think a decentralized open source remote desktop app should at least mirror Twitter posts on a decentralized open source social platform.  
The RustDesk users already arrived at Mastodon ;-)  
https://mastodon.social/tags/RustDesk  
https://mastodon.social/@rust_discussions/109835635585914557  
https://mastodon.social/@unfa/109007904903299238  
https://en.wikipedia.org/wiki/Mastodon_(social_network)

In case you don't like to click Twitter links as I do, here's the tweets content:  
https://twitter.com/rustdesk/status/1698372220379349421  
_2023-09-03_  
_(at)rustdesk(at)twitter.com_  
_We have temporarily unpublished RustDesk from Google Play to prevent further scams targeting users._
https://twitter.com/rustdesk/status/1697615432465723661  
_2023-09-01_  
_(at)rustdesk(at)twitter.com_  
_#scammers is going to destroy us. We may have to pull down public servers._  
![grafik](https://github.com/rustdesk/rustdesk/assets/3355089/ed3994e9-2b10-4ec5-8c4b-28b831d2db02)

**Reply by My1** [reply]
is it allowed to upload your own rustdesk version nailed to your own server to the playstore?

**Comment by Philippe34** [comment]
I don't understand how the scammers' modus operandi couldn't also apply to other remote control applications. A scammer calling you to ask you to give him access to your machine can be done with any of the other applications.

**Reply by JokerGermany** [reply]
yeah i don't think it's a good idea to remove the App from the App Store...
Scammers will use another way...
Now i have to go back to Teamviewer.... Nice ...

**Reply by GEOEGII555** [reply]
Well, scammers were actively using TeamViewer. TeamViewer started showing alerts when you connect from India now tho.

**Comment by endlessterror** [comment]
Responding to users' comments on Google Play definitely takes time. Ways to handle this:
1. Throw a huge notice on each app start, advising on scammers. "This application can provide others with access to your phone if YOU allow it. Make sure that you do not do so, unless you actually know who you are talking to. Press here if you fully understand the risks in doing so! <acceptance button>"
2. Make the incoming-access-to-phone a paid-for feature (and also display a similar notice "if someone asked you to install this blah blah").
3. Organise very carefully the application information in google play to indicate the danger of having such a thing.
4. In the same line of thought, make sure to respond timely to users' feedback on the store.

Really it's a pity this app is not on the store... Next time it is, I'll definitely 5-star it (didn't have the chance).

**Reply by My1** [reply]
would be interesting how anydesk exactly handles this. is access to phone paid?

so far what I know are that they changed the default profile to view only, and additionally gives a warning with a 10 second timeout on connect

**Comment by CrazyBoyFeng** [comment]
It looks like it's because rustdesk gets a lot of complaints about being used for scams.
My guess is that teamviwer and anydesk they are not often complained about for scams because they have to use public servers where scammers can be easily traced. And they have limits on the number and frequency of connections. In addition they don't offer password inbound by default on their mobile, only through an authorization window.
Borrowing from their strategy, I think rustdesk can consider the following improvement programs:
1. modify the client to only accept connection requests from official relay servers if no third-party relay servers are set up by default. This will stop fraudsters from using private relay servers to circumvent traceability.
2. Limit how often each id and ip connects to other ids on the official server. Too many clients connected in a short period of time means it is either a scammer or a professional user. For professional users, there is a good reason to require it to use a professional version. There may be some professional users who find it cumbersome to write private servers to each client. In addition to letting users compile the client with their own code changes, the development team at rustdesk might want to consider launching a free or paid customized client service.
3. mobile does not provide one-time password inbound by default, if you want to use one-time password inbound, you have to go into the settings to enable it manually. 
4. prompt users in the authorization window about the risk of authorizing screen sharing and remind them not to authorize to strangers. Customer service generally does not use remote control to serve consumers, and if the other party claims to be customer service, then it is most likely a scam.

In addition, in order to salvage rustdesk's reputation, rustdesk may need to go to great lengths to explain to complaining users that rustdesk is not malware, but merely a remote control software used for friends and family assistance or for office work, and to advice the victim to seek help from his or her bank or even the police.

**Comment by NintendoManiac64** [comment]
Regarding the idea of warning users about scams or the like, I'm reminded of how the Wii soft-mod "hackmii installer" for installing the homebrew channel and/or bootmii throws up a huge full-screen warning regarding scams.

Basically the below warning screen actually sits there for something like 30 seconds to a minute before the "press 1 to continue" even appears to then allow you to continue to the next screen (though there is actually background processing so there is at least a practical excuse for forcing the user to wait).

![Hackmii installer ScamScreen](https://github.com/rustdesk/rustdesk/assets/11896003/6abc62c2-03da-44db-bd33-964a90277fc4)

**Reply by NintendoManiac64** [reply]
It would seem that such a thing is already sort of implemented in v1.2.3.

...but I say "sort of" because I noticed that the warning does not appear if you instead directly enable "screen capture" which then subsequently enables the "start service" without giving the SCAM warning.

**Comment by JaCh0o** [comment]
Nevertheless will it be restored to Google Play Store since this is main and default app market for android devices such as app store for apple devices? 

Moreover I got issue with it because Norton 360 android app is marking this app as destructive (while anydesk or TeamViewer not)

**Comment by phormix** [comment]
Yeah, if the goal (for personal use) is to have an easy, self-hosted means of supporting friends and family who may be less-savvy, then "download this APK, click through a bunch of stuff to enable it,  etc etc" is going to be a lot more headache than "download thing from app store, connect to X"

It also means that getting updates is no longer an automatic function.

Having the big timed warning about scams at first connection to an external server seems like it would be a legitimate way to reduce the issue while still allowing it to be installed and maintained from a trusted source and

**Reply by JokerGermany** [reply]
I am installing every user [neostore](https://f-droid.org/de/packages/com.machiav3lli.fdroid/) now, [rustdesk is available at f-droid](https://f-droid.org/de/packages/com.carriez.flutter_hbb/)

**Comment by MichaelUray** [comment]
When will the RustDesk App be available again in the Google Play store?

**Reply by My1** [reply]
Considering google play protect flags rustdesk down, it'll likely need some changes first

**Reply by xlionjuan** [reply]
https://github.com/rustdesk/rustdesk/discussions/7109#discussioncomment-8853725

https://github.com/rustdesk/rustdesk/discussions/7109#discussioncomment-8851591

**Comment by EntityinArray** [comment]
This decision also makes app pointless for helpdesk support, because you can't expect non tech-savvy person to install F-Droid or an .apk when they need help.

**Reply by Poordeveloper** [reply]
What a selfish and self-righteous point.

Do you really think the RustDesk team doesn't know what you know? You only care about yourself and never care about whose elderly parents get scammed. You save 1 cent, but someone may lose their life because of getting scammed.

**Reply by Cazman80** [reply]
@Poordeveloper bit of a harsh reply...
Yes the decision to take it down is probably the right one, until they can figure out a way to protect people better...
But if we take your stance on this, we should get rid of RustDesk all together shouldn't we? I don't think the warning banners on the website would be that effective.

@EntityinArray point is true, you can't expect end users to install an APK which makes RustDesk not viable to support android phones, hopefully they figure out a way to protect users better and get the app back in the play store.

But jumping to the conclusion so passionately that he only cares about himself, and that his decision may lead to people losing their life to save 1 cent is really poor.
From his profile, he is fairly active on Github, much more than you, I'd say that's a fair indicator that he is giving to the community and not just caring about himself.

**Reply by rustdesk** [reply]
I guess I need to lock the discussion, because it is meaningless. No one can persuade others, everyone is standing on their own position.
